packages/backend/test/peripherals/database/UserRegistrationEventRepository.test.ts

```diff
@@ -139,4 +139,34 @@ describe(UserRegistrationEventRepository.name, () => {
     const event = await repository.findByStarkKey('0x123')
     expect(event).not.toBeDefined()
   })
+
+  it('finds event by ethereum address', async () => {
+    const event = {
+      blockNumber: 2,
+      ethAddress: EthereumAddress.random(),
+      starkKey: '0x1235',
+    }
+
+    const records = [
+      {
+        blockNumber: 1,
+        ethAddress: EthereumAddress.ZERO,
+        starkKey: '0x1234',
+      },
+      event,
+    ]
+
+    await repository.add(records)
+    const actual = await repository.findByEthereumAddress(event.ethAddress)
+
+    expect(actual).toEqual({ id: 2, ...event })
```

Comment:
This test is failing with id being `11` and not `2`. Are the ids here even something we're concerned about? I think in other places we use `expect.a(Number)`.

I also checked that this test was failing before I merged the master branch into this branch.